base_layer/core/tests/node_service.rs

```diff
@@ -288,35 +298,39 @@ fn propagate_and_forward_valid_block() {
     let bob_node_identity = random_node_identity();
     let carol_node_identity = random_node_identity();
     let dan_node_identity = random_node_identity();
-    let (block0, _) = create_genesis_block(&factories);
-    let consensus_constants = ConsensusConstants::current_with_network(Network::LocalNet(Box::new(block0.clone())));
-    let rules = ConsensusManager::new(None, consensus_constants);
-    let mut alice_node = BaseNodeBuilder::new()
+    let network = Network::LocalNet;
+    let consensus_constants = network.create_consensus_constants();
+    let (block0, _) = create_genesis_block(&factories, &consensus_constants);
+    let rules = ConsensusManagerBuilder::new(network)
+        .with_consensus_constants(consensus_constants)
```

Comment:
Can delete this line (and everywhere else you just use the default constants, so I'll stop commenting now :) )

Reply:
done